lib/msal-core/src/AuthResponse.ts

```diff
@@ -18,3 +18,19 @@ export type AuthResponse = {
     account: Account;
     accountState: string;
 };
+
+export function buildResponseStateOnly(state: string) : AuthResponse {
+    let response : AuthResponse = {
```

Comment:
could just 
```js
return { 
  fields
};
```

instead of defining a var you are not using

Reply:
updated, resolving comments. unresolve if you feel they arent addressed.